src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
+    public void draw();

```

Comment:
```suggestion
    void draw();
```
interface methods are public be default